v1.7/ics/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: IBM
+name: IBM Container Service
+version: v1.7.4
+website_url: https://www.ibm.com/Bluemix/Containers
+documentation_url: https://console.bluemix.net/docs/containers/container_index.html
+product_logo_url: http://ibm.biz/containerslogo
```

Comment:
Hopefully it is just me, but http://ibm.biz/containerslogo is coming back with a blank screen.